.travis.yml

```diff
@@ -9,8 +9,7 @@ php:
   - 7.1
   - hhvm
 
-before_script:
-  - composer install --prefer-source
+install: travis_retry composer install
```

Comment:
Half of the Travis scripts we have contain this "--prefer-source" parameter, half of them don't. But I don't know why, so I went for the more simple, more obvious thing and removed it.